commands/theme/preview.ts

```diff
@@ -62,24 +76,26 @@ const handleUserInput = () => {
       onTerminate();
     }
   });
-};
-
-const determineSrcAndDest = async options => {
+}
+async function determineSrcAndDest(args: ThemePreviewArgs): Promise<{
+  absoluteSrc: string;
+  dest: string;
+}> {
   let absoluteSrc;
   let dest;
   const { projectDir, projectConfig } = await getProjectConfig();
   if (!(projectDir && projectConfig)) {
     // Not in a project, prompt for src and dest of traditional theme
-    const previewPromptAnswers = await previewPrompt(options);
-    const src = options.src || previewPromptAnswers.src;
-    dest = options.dest || previewPromptAnswers.dest;
+    const previewPromptAnswers = await previewPrompt(args);
+    const src = args.src || previewPromptAnswers.src;
+    dest = args.dest || previewPromptAnswers.dest;
     absoluteSrc = path.resolve(getCwd(), src);
     if (!dest || !validateSrcPath(absoluteSrc)) {
       process.exit(EXIT_CODES.ERROR);
     }
   } else {
     // In a project
-    let themeJsonPath = getThemeJSONPath();
+    let themeJsonPath = getThemeJSONPath(getCwd());
```

Comment:
`getThemeJSONPath()` requires one argument--a path. The path here is the CWD. If you look above, `getProjectConfig` searches for the project config in the CWD if no argument is provided.